standards/core/lightpush.md

```diff
@@ -0,0 +1,86 @@
+---
+title: WAKU2-LIGHTPUSH
```

Comment:
Small comment: I think we should start only using the "waku" term. I see is actually well used in line 23.
```suggestion
title: WAKU-LIGHTPUSH
```